### PR TITLE
fixed testing issue

### PR DESCRIPTION
**Pull request by Shivraj0**

### CODE REVIEW

File: src/lib/Theme.ts [hunk 1]
```diff
@@ -87,6 +88,7 @@ export default class Theme {
         } catch(err) {
             Logger.error(`Error reading ${path.split('/').slice(-1)[0]} file.!!!`);
             Logger.error(err);
+            process.exit(1);
```
Comment (vivek8690) [inline src/lib/Theme.ts]: Handle this centrally

File: src/lib/Theme.ts [hunk 2]
```diff
@@ -75,6 +75,7 @@ export default class Theme {
         } catch(err) {
             Logger.error(`Error writing ${path.split('/').slice(-1)[0]} file.!!!`);
             Logger.error(err);
+            process.exit(1);
```
Comment (vivek8690) [inline src/lib/Theme.ts]: Handle this centrally

**Review by vivek8690 — CHANGES_REQUESTED** [review]
Reviewed

File: src/lib/Theme.ts (hunk 1 above)
Comment (Shivraj0) [reply src/lib/Theme.ts]: Done

File: src/lib/Theme.ts (hunk 2 above)
Comment (Shivraj0) [reply src/lib/Theme.ts]: Done

**Review by vivek8690 — APPROVED** [review]

**Pull request merged**